adyen/assets/logs/adyen.yaml

```diff
@@ -0,0 +1,83 @@
+id: adyen
+metric_id: adyen
+backend_only: false
+facets:
+  - groups:
+      - Event
+    name: Event Name
+    path: evt.name
+    source: log
+pipeline:
+  type: pipeline
+  name: Adyen
+  enabled: true
+  filter:
+    query: "source:adyen"
+  processors:
+    - type: string-builder-processor
```

Comment:
I've noticed the use of `string-builder-processor` (here and below) instead of using a simple attribute remapper.
Was this done on purpose?
I imagine this can be related to the fact that notificationItems is an array but I'd like to confirm.

Reply:
Yes, we used `string-builder-processor` because `notificationItems` is an array.